dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -83,4 +90,43 @@ private static File findLocalRepository() {
    }
   }
 
+  /**
+   * Parse the dependencyManagement section of an artifact and return the
+   * artifacts included there.
+   */
+  // TODO Need to consider the
+  // possibility that the artifact is not a BOM; that is, that it does not have
+  // a dependency management section.
+  public static List<Artifact> readBom(Artifact artifact) throws ArtifactDescriptorException {
+    RepositorySystem system = RepositoryUtility.newRepositorySystem();
+    RepositorySystemSession session = RepositoryUtility.newSession(system);
+
+    ArtifactDescriptorRequest request = new ArtifactDescriptorRequest();
+    request.addRepository(RepositoryUtility.CENTRAL);
+    request.setArtifact(artifact);
+
+    ArtifactDescriptorResult resolved = system.readArtifactDescriptor(session, request);
+    List<Exception> exceptions = resolved.getExceptions();
+    if (!exceptions.isEmpty()) {
+      throw new ArtifactDescriptorException(resolved, exceptions.get(0).getMessage());
+    }
+    
+    List<Artifact> managedDependencies = new ArrayList<>();
+    for (Dependency dependency : resolved.getManagedDependencies()) {
+      Artifact managed = dependency.getArtifact();
+      // TODO remove this hack once we get these out of 
+      // google-cloud-java's BOM
+      if (managed.getArtifactId().equals("google-cloud-logging-logback")
+          || managed.getArtifactId().equals("google-cloud-contrib")) {
+        continue;
+      }
+      if (!managedDependencies.contains(artifact)) {
+        managedDependencies.add(managed);
+      } else {
+        System.err.println("Duplicate dependency " + dependency);
+      }
+    }
+    return managedDependencies;
+  }
```

Comment:
Since this section moved files, I can't tell what changed - could you point it out?